src/mappings/comptroller.ts

```diff
@@ -29,7 +37,12 @@ export function handleMarketEntered(event: MarketEntered): void {
 }
 
 export function handleMarketExited(event: MarketExited): void {
-  let market = Market.load(event.params.cToken.toHexString())
+  log.info('COMPTROLLER::handleMarketExited', [])
+  let marketId = event.params.cToken.toHexString()
+  let market = Market.load(marketId)
+  if (market === null) {
```

Comment:
why are we adding this extra `market==null` check everywhere, it is always not null right (acc to compound code). 
Did we have any instance where market was null and it was breaking?

Reply:
yes, there were some instance where it was coming back as null and breaking